pyxrf/model/fileio.py

```diff
@@ -585,43 +662,60 @@ def output_data_to_tiff(fit_output,
         tiff or txt
     name_append: str, optional
         more information saved to output file name
-    norm_name : str, optional
+    scaler_name : str, optional
         if given, normalization will be performed.
     use_average : Bool, optional
-        when normalization, multiply mean value of denomenator,
-        i.e., norm_data = data1/data2 * np.mean(data2)
+        when normalization, multiply by the mean value of scaler,
+        i.e., norm_data = data/scaler * np.mean(scaler)
     """
+
+    if output_folder is None:
+        raise ValueError("Output directory is not specified.")
+
+    if name_append:
+        name_append = f"_{name_append}"
+
+    file_format = file_format.lower()
+    # The following works only if 'file_format' is specified by extension ('txt', 'tiff', etc.)
```

Comment:
I'd be more protective to avoid "surprises" in the future. Since you mentioned the allowed values in the docstring, do you mind to replace the above comment with the actual code, such as in the suggestion below?
```suggestion
    allowed_formats = ('txt', 'tiff')
    assert file_format in allowed_formats, f"The specified format '{file_format}' not in {allowed_formats}"
```

Reply:
Thanks!